docs/dev_guide/frequently-asked-questions-faq.rst

```diff
@@ -336,5 +336,10 @@ is adjusted to avoid a bit shift that is greater than the type length:
      ...
    }
 
+**How do I resolve missing include errors that occur when migrating my code?**
 
-
+Use the option ``--extra-arg=-v`` to prompt |tool_name| to use verbose
+output, which includes information about which paths the tool searches
+for includes. To add an additional path for the tool to use when
+searching for includes during migration, use the option
+``--extra-arg="-I<extra include path>"`` in your migration command.

```

Comment:
For user who migrate a project with compilation data base, the end user may udpate compilation db to include extra path.